src/Game/UI/Gumps/ContainerGump.cs

```diff
@@ -193,6 +201,14 @@ public override void Update(double totalMS, double frameMS)
                 return;
             }
 
+            if (item.Items.Count() > 0)
```

Comment:
.Count() is very expensive in terms of performance.
Use .Count property instead.

Reply:
Good catch!
I will fix once I get some time over.
Or feel free to edit yourself. :)

Reply:
Fixed